akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -151,11 +118,9 @@ object Sink extends SinkApply {
    * element is dropped and the stream continues.
    *
    * @see [[#mapAsyncUnordered]]
-   */
+   */ // FIXME Why is this method even useful?
```

Comment:
Sorry for adding this comment, but it was the only way for me to remember to write this comment:

What is the purpose of having this operation as a prime API construct when `mapAsyncUnordered` + `Sink.ignore` is expressive enough for the (unknown) use-case?